src/components/Articles.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from "react";
+import { fetchArticles } from "../utils/api";
+
+const Articles = () => {
+  const [articleList, setArticleList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchArticles().then(articles => {
```

Comment:
You might want to setIsLoading to `true` here before you fetch articles, just in case the page needs to be rerendered in the future!